neuralprophet/forecaster.py

```diff
@@ -293,7 +293,8 @@ def __init__(
         optimizer="AdamW",
         newer_samples_weight=2,
         newer_samples_start=0.0,
-        quantiles=[0.5],
+        confidence_interval=None,
+        quantiles=None,
```

Comment:
Let's also Introduce a new parameter: `uncertainty_estimation=”quantile_regression”`

Reply:
This is done.

Reply:
Renamed it to `uncertainty_method`.